benchmarks/logic.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from sympy.core.compatibility import range
+from sympy.logic.utilities import load_file
+from sympy.logic import satisfiable
+
+input_path = os.path.dirname(__file__)
+
+
+class LogicSuite:
+    """
+    Benchmark suite for sympy.logic
+    """
+    def setup(self):
+        self.INPUT = [5*i for i in range(2, 16)]
+
+    def time_dpll(self):
+        for test in self.INPUT:
+            file_name = os.path.join(input_path, 'logic-inputs', '%s.cnf' % test)
+            theory = load_file(file_name)
```

Comment:
Maybe all files should be loaded in `setup(self)`? (in the event that loading would be slow)
You could also benchmark the loading a "time_load" which loads a single file.